packages/lumx-react/src/components/badge/Badge.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { mdiHeart } from '@lumx/icons';
+import { AspectRatio, Badge, ColorPalette, Icon, Size, Thumbnail, ThumbnailVariant } from '@lumx/react';
+import { decorators } from '@lumx/react/story-block';
+import { select, text } from '@storybook/addon-knobs';
+import React from 'react';
+
+const options = {
+    Blue: ColorPalette.blue,
+    Dark: ColorPalette.dark,
+    Green: ColorPalette.green,
+    Light: ColorPalette.light,
+    Primary: ColorPalette.primary,
+    Red: ColorPalette.red,
+    Secondary: ColorPalette.secondary,
+    Yellow: ColorPalette.yellow,
+};
+
+export default { title: 'Badge', decorators };
+
+export const badgeValue = () => (
```

Comment:
@gigeos would you mind adding a story name that is a little bit more descriptive ? This name shows up on storybook as "Badge value", which might be a little bit confusing since it is not very descriptive. You can try with names like `simpleBadgeWithValue`. Also it makes it more readable on the Storybook side.